spec/computer_spec.rb

```diff
@@ -1,8 +1,32 @@
 require "computer"
+require "spec_helper"
+require "board"
 
 RSpec.describe Computer do
-  it "creates a new computer with a mark" do
-    computer = Computer.new("O")
+  board = Board.new(empty_board)
```

Comment:
Looks like you're defining these outside of any scope. should either initialise in a setup method, or use let(:board) { Board.new } syntax.

Reply:
Resolved #36